db/seedfiles/translations/en/plan.rb

```diff
@@ -0,0 +1,126 @@
+  'en.plans.tty_number' => '711 TTY',
+  'en.plans.bhp_phone' => '1-855-833-8120',
+  'en.plans.bhp_href_phone' => 'tel:+18558338120',
+  'en.plans.hours' => 'Hours:',
+  'en.plans.bhp_hours' => 'Monday through Friday from 8:00 a.m. to 6:00 p.m. Eastern Time',
+  'en.plans.email' => 'Email:',
+  'en.plans.bhp_email' => 'memberquestions@bmchp.org',
+
+  'en.plans.bmchp_phone' => '855-833-8120',
+  'en.plans.bmchp_email' => 'memberquestions@bmchp.org',
+  'en.plans.bmchp.hours' => 'Monday through Friday from 8:00 a.m. to 6:00 p.m.',
+
+  'en.plans.fh_phone' => '800-868-5200',
+  'en.plans.fh_href_phone' => 'tel:+18008685200',
+  'en.plans.fh_tty_number' => 'Please call TRS Relay 711.',
+  'en.plans.fh_email' => 'FCHPcustomerservice@fallonhealth.org',
+  'en.plans.fh_hours' => 'Monday, Tuesday, Thursday, and Friday from 8:00 a.m. to 6:00 p.m., and Wednesday from 10:00 a.m. to 6:00 p.m.',
+
+  'en.plans.local' => 'Local:',
+  'en.plans.toll_free' => 'Toll-free:',
+  'en.plans.ttd/tty' => 'TTY/TDD:',
+  'en.plans.hne_local_phone' => '413-787-4004',
+  'en.plans.hne_href_local' => 'tel:+14137874004',
+  'en.plans.hne_toll_free' => '800-310-2835',
+  'en.plans.hne_href_toll' => 'tel:+18003102835',
+  'en.plans.hne_tty' => '800-439-2370',
+  'en.plans.hne_href_tty' => 'tel:+18004392370',
+  'en.plans.hne_email' => 'memberservices@hne.com',
+  'en.plans.hne_hours' => 'Monday through Friday, 8:00 a.m. to 6:00 p.m.',
+
+  'en.plans.bcbs_ivl' => 'For Individual/Non-Group:',
+  'en.plans.bcbs_small' => 'For Small Group/FOM:',
+  'en.plans.bcbs_ivl_phone' => '1-800-822-2700',
+  'en.plans.bcbs_href_ivl_phone' => 'tel:+18008222700',
+  'en.plans.bcbs_small_phone' => '1-800-782-3675',
+  'en.plans.bcbs_small_href_phone' => 'tel:+18007823675',
+  'en.plans.bcbs_tty_number' => '711',
+  'en.plans.bcbs_email' => 'send secure email after login to member account',
+  'en.plans.bcbs_hours' => 'Monday through Friday, 8:00 a.m. to 6:00 p.m. Eastern Time',
+
+  'en.plans.hphc_phone' => '888-333-4742',
+  'en.plans.hphc_href_phone' => 'tel:+18883334742',
+  'en.plans.hphc_tty' => '711',
+  'en.plans.hphc_email' => 'send secure email after login to member account',
+  'en.plans.hphc_hours' => 'Monday, Tuesday, & Thursday from 8:00 a.m. to 6:00 p.m.; Wednesday from 10:00 a.m. to 6:00 p.m.; and Friday from 8:00 a.m. to 5:30 p.m',
+
+
+  'en.plans.ahp_phone' => '866-414-5533',
+  'en.plans.ahp_href_phone' => 'tel:+18664145533',
+  'en.plans.ahp_tty' => '711',
+  'en.plans.ahp_email' => 'customerservice@AllWaysHealth.org',
+  'en.plans.ahp_hours' => 'Monday, Tuesday, Wednesday, Friday from 8:00 a.m. to 6:00 p.m.; Thursday from 8:00 a.m. to 8:00 p.m.',
+
+
+  'en.plans.uh_phone' => '1-888-842-4571',
+  'en.plans.uh_href_phone' => 'tel:+18888424571',
+  'en.plans.uh_tty' => '711',
+  'en.plans.uh_hours' => '7 AM to 6 PM CST',
+
+
+  'en.plans.thd_phone' => '888-257-1985',
+  'en.plans.thd_href_phone' => 'tel:+18004620224',
+  'en.plans.thd_tty' => '711',
```

Comment:
can you globalize this and us the same key?

Reply:
@sri49 some carriers have same tty, and some have different tty number. They tend to change as they were changed this year. I would advise to leave it instead of globalize this.